ci-operator/step-registry/rhcs/e2e/setup/rhcs-e2e-setup-commands.sh

```diff
@@ -77,6 +77,12 @@ fi
 if [ ! -z "$RHCS_VERSION" ]; then
     export RHCS_VERSION=$RHCS_VERSION
 fi
+if [ ! -z "$RHCS_CLUSTER_NAME_PREFIX" ]; then
+    export RHCS_CLUSTER_NAME_PREFIX=$RHCS_CLUSTER_NAME_PREFIX
+fi
+if [ ! -z "$RHCS_CLUSTER_NAME_SUFFIX" ]; then
+    export RHCS_CLUSTER_NAME_SUFFIX=$RHCS_CLUSTER_NAME_SUFFIX
+fi
```

Comment:
Actually, we do not need to `export` the pod's environment variable again unless we want to do more around it.

Reply:
I was not sure about that